Adaptors/Amqp/src/PullQueueStorage.cs

```diff
@@ -90,8 +91,10 @@ public class PullQueueStorage : QueueStorage, IPullQueueStorage
     logger_ = logger;
   }
 
-  public new ValueTask<bool> Check(HealthCheckTag tag)
-    => ValueTask.FromResult(isInitialized_);
+  public new Task<HealthCheckResult> Check(HealthCheckTag tag)
+    => Task.FromResult(isInitialized_
```

Comment:
I would expect a test a bit more involved here.

Reply:
I took the initial one. Most of them are like that. Moreover this health check is not registered.

Reply:
I will not modify this adaptor too much because of https://github.com/aneoconsulting/ArmoniK.Core/pull/150